<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SNMP_Viewer</class>
 <widget class="QWidget" name="SNMP_Viewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1021</width>
    <height>231</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="LE_Version">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Version</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LE_Community">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Community</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="SPN_PDU">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>30</y>
     <width>44</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="SNMP_PDU_Viewer" name="PGE_PDU" native="true">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>30</y>
     <width>811</width>
     <height>201</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>0</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>PDUs</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LE_NumPDU">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>121</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Number of PDUs</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SNMP_PDU_Viewer</class>
   <extends>QWidget</extends>
   <header>snmp_pdu_viewer.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
